FreestarAds-Ogury/Source/objc/OguryBannerMediator.m

```diff
@@ -0,0 +1,119 @@
+//
+//  OguryBannerMediator.m
+//  FreestarAds-Ogury
+//
+//  Created by Carlos Alcala on 7/04/22.
+//
+
+#import "OguryBannerMediator.h"
+#import <FreestarAds/FreestarAds.h>
+#import <UIKit/UIKit.h>
+#import <OgurySdk/Ogury.h>
+#import <OguryAds/OguryAds.h>
+#import <OguryChoiceManager/OguryChoiceManager.h>
+
+@interface OguryBannerMediator()<OguryBannerAdDelegate, FSTRMediatorEnabling>
+
+@property (nonatomic, strong) OguryBannerAd *ad;
+@property OguryAdsBannerSize *requestedSize;
+@property CGFloat requestedWidth;
+@property(nonatomic, assign) FreestarBannerAdSize freestarAdSize;
+
+
+@end
+
+@implementation OguryBannerMediator
+
+- (BOOL)isEnabledForMediatorFormat:(FSTRMediatorFormatType)type {
+    switch (type) {
+        case FSTRMediatorFormatTypeBanner:
+            return YES;
+        default:
+            return NO;
```

Comment:
Does ogury support inline (mrec)? if so, we need to enable it here.

Reply:
Sure will enable